addons/website/static/src/client_actions/website_preview/website_preview.scss

```diff
@@ -75,6 +75,7 @@
             height: 0px;
             transform: translateY(-$o-navbar-height);
             border-bottom: 0px;
+            padding: 0;
```

Comment:
I don't know why this file uses this indentation, but it must be for some reason. Could you keep it?

Reply:
Nevermind, it was on my side, git preview was broken 😅